source/sass/components/article-page.scss

```diff
@@ -403,6 +403,12 @@
   background: rgb(231, 231, 252); // One-off colors; no variable exists for this
   /* stylelint-enable */
   padding: 9px 0;
+  position: sticky;
+  margin-top: -100px;
```

Comment:
We're hiding the nav by giving it a negative margin. Then using CSS to "remove" (reset) the margin so it scrolls down into view. And when the summary bar should be hidden it scroll back up to -100px

Reply:
Let's turn this `-100px` into a custom css properties. e.g., `--margin-top: -100px;` so we can use JS to access the value instead hardcode it to `-100px` in JS.